packages/shorebird_cli/lib/src/executables/flutter_version.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:shorebird_cli/src/shorebird_process.dart';
+
+/// Parses the output of `flutter --version` to get the version number.
+///
+/// The output of `flutter --version` is of the form:
+///
+/// ```text
+/// Flutter 3.32.4 • channel stable • https://github.com/flutter/flutter.git
+/// Framework • revision 6fba2447e9 (5 weeks ago) • 2025-06-12 19:03:56 -0700
+/// Engine • revision 8cd19e509d (5 weeks ago) • 2025-06-12 16:30:12 -0700
+/// Tools • Dart 3.8.1 • DevTools 2.45.1
+/// ```
+String? maybeParseFlutterVersionFromOutput(String output) {
```

Comment:
I still think we should consolidate this with `ShorebirdFlutter`. The intention of `ShorebirdFlutter` was to interface with the underlying Flutter installation. If we merged this we would have `flutter --version` execution and parsing in multiple places.